server/model/monitor.js

```diff
@@ -971,7 +982,13 @@ class Monitor extends BeanModel {
             } else if (bean.status === MAINTENANCE) {
                 log.warn("monitor", `Monitor #${this.id} '${this.name}': Under Maintenance | Type: ${this.type}`);
             } else {
-                log.warn("monitor", `Monitor #${this.id} '${this.name}': Failing: ${bean.msg} | Interval: ${beatInterval} seconds | Type: ${this.type} | Down Count: ${bean.downCount} | Resend Interval: ${this.resendInterval}`);
+                let groupText = "";
```

Comment:
What do you think about this variable name?

```suggestion
                let isUnderGroup = "";
```

Reply:
I'll stay with groupText.